x-pack/plugins/ml/public/file_datavisualizer/components/import_view/importer/csv_parse.js

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+
+// promisifying the csv-parse parse function
+
+import csvParse from 'csv-parse';
+
+export function parse(data, settings) {
+  return new Promise((resolve, reject) => {
+    csvParse(data, settings
+      , (err, out) => {
```

Comment:
Any reason why this is formatted with the comma on a new line, rather than all on the one line. Looks a bit odd as it is.